citeproc/source/json.py

```diff
@@ -3,13 +3,12 @@
                         unicode_literals)
 from citeproc.py2compat import *
 
-import sys
 import unicodedata
 
 from . import BibliographySource, Reference
 from . import Pages, Name, Date, DateRange, LiteralDate
 from ..string import String, MixedString, NoCase
-from .. import NAMES, DATES, NUMBERS
```

Comment:
unused.  if there is a possibility that a client might be using this, I can add an __all__ with NUMBERS in it.